.devcontainer/devcontainer.json

```diff
@@ -7,7 +7,8 @@
 			// To ensure that the group ID for the docker group in the container
 			// matches the group ID on the host, add this to your .bash_profile on the host
 			// 		export DOCKER_GROUP_ID=$(getent group docker | awk -F ":" '{ print $3 }')
-			"DOCKER_GROUP_ID": "${localEnv:DOCKER_GROUP_ID:-1001}"
+			// "DOCKER_GROUP_ID": "${localEnv:DOCKER_GROUP_ID:-1001}"
```

Comment:
May as well remove this

Reply:
oops - included by mistake. Lemme add the rest of the docker stuff we discussed to make amends!